rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -1033,7 +1042,7 @@ SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 3" "id:932016,phase:2,pass,nolog,skipAf
 #   util/regexp-assemble/regexp-assemble.py update 932106
 #
 # This rule is a stricter sibling of rule 932100.
```

Comment:
Update mentions of 932100 in comments of other rules.

Reply:
Removed.